setup.cfg

```diff
@@ -34,4 +34,12 @@ disable=
 
 [pylint.MISCELLANEOUS]
 ; http://pylint.pycqa.org/en/latest/technical_reference/features.html#miscellaneous-checker-options
-notes = FIXME
\ No newline at end of file
+notes = FIXME
+
+
+[mypy]
+ignore_missing_imports = True
```

Comment:
My proposition:
1. Configuration + running `mypy setup.py lbz` (121 errors in 12 files) - the first pull request
2. Running `mypy examples tests` - the second pull request
3. `flake8` - the third pull request

The order of PRs matters because `tests` and `examples` depends on the main code - you won't fix all errors without touching the main code first 😉 `flake8` should be done separately, even if doesn't require too much work (if you want, you can do this even as a first step).